src/dev/build/tasks/os_packages/docker_generator/templates/Dockerfile

```diff
@@ -49,11 +49,12 @@ EXPOSE 5601
 
   # crypto-policies not currently compatible with libnss :sadpanda:
   RUN printf "[main]\nexclude=crypto-policies" >  /etc/dnf/dnf.conf
+
+  # update microdnf to have exclusion feature for dnf configuration
+  {{packageManager}} update microdnf --setopt=tsflags=nodocs -y
```

Comment:
We still need this to be in a retry.

Reply:
Want to just wrap it in `{{#ubi}}` where it was?

Reply:
👍

Reply:
@tylersmalley done